webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -29,47 +29,86 @@
     <template #content>
       <v-card flat>
         <div v-if="audienceTypesDisplay" class="mt-1 px-4 pt-4">
-          <span class="subtitle-1 text-color"> {{ $t('activity.composer.content.title') }} </span>
-          <v-radio-group
-            v-if="postToNetwork"
-            v-model="audienceChoice"
-            class="mt-0"
-            mandatory>
-            <v-radio value="yourNetwork">
-              <template #label>
-                <span class="text-color text-subtitle-2 ms-1"> {{ $t('activity.composer.content.yourNetwork') }}</span>
-              </template>
-            </v-radio>
-            <v-radio value="oneOfYourSpaces">
-              <template #label>
-                <span class="text-color text-subtitle-2 ms-1"> {{ $t('activity.composer.content.oneOfYourSpaces') }}</span>
-              </template>
-            </v-radio>
-          </v-radio-group>
-          <exo-identity-suggester
-            v-if="spaceSuggesterDisplay"
-            ref="audienceComposerSuggester"
-            v-model="audience"
-            :labels="spaceSuggesterLabels"
-            :include-users="false"
-            :width="220"
-            name="audienceComposerSuggester"
-            class="user-suggester mt-n2"
-            include-spaces
-            only-redactor />
-          <div v-else-if="audience && postInYourSpacesChoice">
-            <v-chip
-              class="primary"
-              close
-              @click:close="removeAudience">
-              <v-avatar left>
-                <v-img :src="audience.profile.avatarUrl" role="presentation" />
-              </v-avatar>
-              <span class="text-truncate">
-                {{ audience.profile.fullName }}
-              </span>
-            </v-chip>
+          <div v-if="(postInYourSpacesChoice && !audience) || postInYourNetwork">
```

Comment:
May be add this in a dedicated computed param, WDYT ?

Reply:
yess, done!